.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - android/save-gradle-cache
       - android/save-test-results
   Lint:
-    executor: 
+    executor:
```

Comment:
I opened the file to look for the step that copies the debug keystore like WordPress and Woo do, but couldn't find it.

My editor removed these trailing whitespaces and I thought I'd live the change in for the better 😄